### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kounocom**
This PR fixes wording, spelling and punctuation in some pages, updates the manual mounting image in quick-setup, and adds info on the BMI270 IMU where it was previously missing

### CODE REVIEW

File: src/diy/components-guide.md
Comment (ButterscotchV) [inline src/diy/components-guide.md:105]:
```suggestion
Choosing or designing a case is where you will have the most freedom to choose whatever you like. If you have access to a 3D printer, you can print some cases for about ~$10 worth of filament. Resin printed cases are not recommended due to their brittle nature. There are some great [Community-built cases](cases.md), likewise it's worth checking the #diy and #cases channels of our [Discord](https://discord.gg/SlimeVR).
```

File: src/assets/img/rotation_bmi270.png
Comment (ButterscotchV) [inline src/assets/img/rotation_bmi270.png:1]: My eyes

File: src/firmware/setup-and-install.md
Comment (ButterscotchV) [inline src/firmware/setup-and-install.md:70]: This feels like it makes the sentence too long, maybe it can be organized a little better?

File: src/quick-setup.md
Comment (ButterscotchV) [inline src/quick-setup.md:41]:
```suggestion
1. Once you have connected all your trackers, you should see them listed with numbers on the right. If you have forgotten which trackers have yet to be plugged in, shaking a powered on tracker will highlight it in the list if it has already been connected. Click **I connected all my trackers** once you are finished.
```

File: src/quick-setup.md
Comment (ButterscotchV) [inline src/quick-setup.md:33]:
```suggestion
1. Input your 2.4GHz Wi-Fi credentials, so that your tracker can connect to Wi-Fi, then click **Submit**.
```

File: src/quick-setup.md
Comment (ButterscotchV) [inline src/quick-setup.md:66]: This isn't necessarily wrong, but should we recommend waist? I've found hip performs better in general, but this is kinda a personal opinion thing when it comes to this.

File: src/quick-setup.md
Comment (ButterscotchV) [inline src/quick-setup.md:121]: Weird wording, assumes the user has done something they may not have done
```suggestion
	* **Tapping Right Thigh** - Mounting reset, resets the trackers to an estimated mounting direction. You must be in the ski position as illustrated in the mounting calibration wizard for this to work.
```

File: src/server/imu-calibration.md
Comment (ButterscotchV) [inline src/server/imu-calibration.md:44]: We can link there :3
```suggestion
If you have a BMI270 tracker with l0ud/main firmware, you will need to calibrate your IMU the same way as you would calibrate a [BMI160 tracker with firmware v0.3.3 and above](#bmi160-with-firmware-v033-and-above).
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]
It's also questionable about "mounting direction", as we still use mixed terms throughout the text and there was never a decision made on the best term. I personally like "mounting orientation", but I'll try to get approval on that before asking it to be changed here.

File: src/quick-setup.md
Comment (kounocom) [reply src/quick-setup.md:66]: I do agree that hip performs better, but there were people in the Discord who struggled to get good mounting on hip and weren't aware that you can use waist instead

Comment (ButterscotchV) [reply src/quick-setup.md:66]: Understandable, I just think people might be attracted towards using "waist" instead as they know "waist tracking" from SteamVR stuff. Either way this is fine, I'll just leave this up for consideration but this isn't really a big issue.

File: src/quick-setup.md
Comment (kounocom) [reply src/quick-setup.md:41]: Yeah I couldn't find a way nice to word it when I was working on this, this is more clear though

File: src/assets/img/rotation_bmi270.png
Comment (kounocom) [reply src/assets/img/rotation_bmi270.png:1]: I basically just overlaid BMI270 on the BMI160 orientation image. Maybe both should be made less crunchy?

Comment (ButterscotchV) [reply src/assets/img/rotation_bmi270.png:1]: Maybe like outline it or smth

File: src/firmware/setup-and-install.md
Comment (kounocom) [reply src/firmware/setup-and-install.md:70]: I've seen a few people in the diy channel clone the MPU+QMC fork even when they don't need it, so I agree and believe it can cause some confusion.

**Comment by ButterscotchV** [comment]
Now that https://github.com/SlimeVR/SlimeVR-Server/pull/939 is merged, I would request "mounting direction" be changed to "mounting orientation" now. I'm also still really iffy about the one remaining comment on my review, idk if I want it merged like that. Please request my review once that's all done and I can approve.

File: src/firmware/setup-and-install.md
Comment (kounocom) [reply src/firmware/setup-and-install.md]: The best way I found to solve this was to just add a step before, explaining how to choose a firmware version based on the hardware the user has.

**Review by ButterscotchV — APPROVED** [review]
Good way of wording it

**Pull request merged**
